examples/next-sse-chat/src/server/railwayPrNumber.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * @example 'trpc-pr-5821'
+ */
+const railwayEnv = process.env.RAILWAY_ENVIRONMENT_NAME;
```

Comment:
RAILWAY_ENVIRONMENT_NAME doesn't seem available here

cc @juliusmarminge can you spot what I'm doing wrong in the Dockerfile or whatever?  

dockerfile logs it but it doesn't reach the build smh

Reply:
You gotta add it to turbo.json to inject it into the script

Reply:
awesomeee thanks